test/setup.js

```diff
@@ -9,39 +9,32 @@
 import fs from 'fs';
 import path from 'path';
 import register from 'babel-core/register';
-import chai from 'chai';
-import chaiEnzyme from 'chai-enzyme';
-import ReactNative from 'react-native-mock';
-import {Sentry} from 'react-native-sentry';
-
-const m = require('module');
-const originalLoader = m._load;
-
-const NativeModules = ReactNative.NativeModules;
-const Platform = ReactNative.Platform;
-
-NativeModules.RNCookieManagerIOS = {};
-NativeModules.RNCookieManagerAndroid = {};
-
-Platform.__setOS('ios');
-
-// Image file ignore setup from:
-// http://valuemotive.com/2016/08/01/unit-testing-react-native-components-with-mocha-and-enzyme/
-m._load = function hookedLoader(request, parent, isMain) {
-    if (request.match(/.jpeg|.jpg|.png$/)) {
-        return {uri: request}
+import mockery from 'mockery';
+// import {Sentry} from 'react-native-sentry';
```

Comment:
This can be removed if it's not needed any more

Reply:
Will do